src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts

```diff
@@ -315,17 +322,18 @@ export class InlineCompletionsAccessibleViewContribution extends Disposable {
 					previous() {
 						model.previous().then(() => show());
 					},
+					onKeyDown: (e) => {
+						if (e.ctrlKey && e.browserEvent.key === '/') {
```

Comment:
This should use a keybinding? Hardcoding keys like this should rarely be done as they can't be customized and there is some nuance with international keyboard

Reply:
I looked into this and while an action would be ideal, we'd need to get the completions controller to not clear the `state` and `model` like it currently does when the accessible view is open. I want to avoid accessibility code in the inline completions code.